pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -802,13 +802,6 @@ def tpu_cores(self) -> Optional[Union[List[int], int]]:
             return self._tpu_cores  # type: ignore
         return 0
 
-    @property
-    def tpu_id(self) -> Optional[int]:
-        if isinstance(self.accelerator, TPUAccelerator):
-            if isinstance(self._tpu_cores, list):
-                return self._tpu_cores[0]
-        return None
-
```

Comment:
Should we also add a deprecation message? Like how it was done [here](https://github.com/PyTorchLightning/pytorch-lightning/pull/12262/files#diff-667a2513c158c2b01d73b479e4dea75587d8531f9cd286891d34570a2fd145dcR2056-R2062).

Reply:
This is being removed directly, instead of being deprecated. The acclerator_connector and its properties are not user facing, they are very PyTorchLightning internal.

Reply:
Thanks, agreed, I didn't find any mention of this in the docs as well. Should be pytorch lightning internal, as you mentioned. Resolving this conversation. :)

Reply:
We could remove it directly